src/main/webapp/app/shared/category-selector/category-selector.component.ts

```diff
@@ -134,10 +133,7 @@ export class CategorySelectorComponent implements OnChanges {
     }
 
     private createCategory(categoryString: string): ExerciseCategory {
-        const category = new ExerciseCategory();
-        category.category = categoryString;
-        category.color = this.chooseRandomColor();
-        return category;
+        return new ExerciseCategory(this.chooseRandomColor(), categoryString);
```

Comment:
Just an idea. But I think this method in general is not really needed as its just called in one place and just creates a new object.

Reply:
It is actually used in two places, so I think the method is reasonable 😅